R/check-devtools.R

```diff
@@ -28,8 +28,8 @@ check_dev_versions <- function(pkg = ".") {
 
   parsed <- lapply(deps$version, function(x) unlist(numeric_version(x)))
```

Comment:
Not that it's a big win in this case, but this could become a `map()`.

Reply:
Done, as well as a few others. I also switched to using `~` for anonymous functions in `map*()` calls

Reply:
I think `~` may not work in the `map_*()` variants offered by the standalone file; that was definitely the case in a previous purrr compat file. You may want to check this, in case the relevant code is not covered by tests.

And, zooming out a bit more, I think we're trying to not create new instances of the `~` style of anonymous function call, as part of the long-term plan to get behind base R's new anonymous function syntax (although we can't use it in packages for a long time).

So, overall, I think it's ok if we use a more verbose method for defining the function that we're mapping.

Reply:
It does work now - I switched to the `~` because I saw this: https://github.com/r-lib/devtools/blob/c9632c423c3e8499d3da94984d2c164702596d4b/R/import-standalone-purrr.R#L34 in the standalone file (and I did test it). But happy to go back to the verbose method!

Reply:
Ah, ok, that changed over time.

But, yeah I think we are trying to not create new usage of the `~`-style syntax. So I think we should do the verbose/explicit method, in the meantime.

Reply:
Sounds good - reverted back to function(x)...